presto-tests/src/test/resources/resource_groups_config_burst.json

```diff
@@ -0,0 +1,49 @@
+{
+  "rootGroups": [
+    {
+      "name": "global",
+      "softMemoryLimit": "1MB",
+      "hardConcurrencyLimit": 100,
+      "maxQueued": 1000,
+      "subGroups": [
+        {
+          "name": "pipeline",
+          "softMemoryLimit": "1MB",
+          "hardConcurrencyLimit": 3,
+          "maxQueued": 3,
+          "schedulingPolicy": "WEIGHTED",
+          "subGroups": [
+            {
+              "name": "scheduled",
+              "softMemoryLimit": "1MB",
+              "softConcurrencyLimit": 2,
+              "hardConcurrencyLimit": 3,
+              "schedulingWeight": 10000,
+              "burstSchedulingWeight": 10,
```

Comment:
is `burstSchedulingWeight` used at all? Also we don't use the term at all in other places (except the test `testBurst`), so that's also confusing.